src/main/resources/application.yaml

```diff
@@ -126,6 +126,11 @@ file-cleanup:
   cron: ${FILE_CLEANUP_CRON:0 15 * * * *}
   ttl: 12h
 
+letter-content-cleanup:
```

Comment:
Maybe it would be worth mentioning somewhere in the name that it's about unposted letters, because we'll still cleanup letter content, just after it's confirm as posted.